components/speedreader/resources/panel/components/toolbar/style.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+
+export const Box = styled.div`
+  display: flex;
+  background: transparent;
```

Comment:
Why does this need a transparent background? By default html elements shouldn't have any background.